chain_start: "base_link" # base frame of commands
chain_end: "tool0" # target eff frame
root_frame: "world" # IK calculation base frame (for multi-follower)

controller: "Velocity"
solver: "MyIK" 


initial_pose: [0.35, 0.0, 0.3, 3.141592, 0, 0]


follower_list: # name space of the robot
  - ""
  # - "toroboarm_2"
  # - "toroboarm_3"
  # - "toroboarm_4"
  # - "toroboarm_5"
  # - "toroboarm_6"
  # - "toroboarm_7"
  # - "toroboarm_8"
  # - "toroboarm_9"

priority: "Manual"  #"Automation" #  (for multi-follower)
MF_mode: "Individual" #"Parallel" #   (for multi-follower)
IK_mode: "Concatenated" # (for multi-follower)
